diff --git a/MODULE.bazel b/MODULE.bazel
index 7723d4c..a15a36b 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -1,6 +1,10 @@
 module(name = "nanobind_example", version = "0.1.0")
 
-bazel_dep(name = "nanobind_bazel", version = "2.4.0")
+bazel_dep(name = "nanobind_bazel", version = "")
+local_path_override(
+    module_name = "nanobind_bazel",
+    path = "../nanobind-bazel",
+)
 bazel_dep(name = "rules_python", version = "1.0.0")
 
 python = use_extension("@rules_python//python/extensions:python.bzl", "python")
diff --git a/src/BUILD b/src/BUILD
index 5562dca..95e210e 100644
--- a/src/BUILD
+++ b/src/BUILD
@@ -20,6 +20,6 @@ nanobind_stubgen(
     name = "nanobind_example_ext_stubgen",
     module = ":nanobind_example_ext",
     marker_file = "src/py.typed",
-    output_directory = "src",
-    recursive = True,
+#    output_directory = "src",
+#    recursive = True,
 )
