diff --git a/MODULE.bazel b/MODULE.bazel
index 191b3c0..402015e 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -1,6 +1,10 @@
 module(name = "nanobind_example", version = "0.1.0")
 
-bazel_dep(name = "nanobind_bazel", version = "2.10.2")
+bazel_dep(name = "nanobind_bazel", version = "")
+local_path_override(
+    module_name = "nanobind_bazel",
+    path = "../"
+)
 bazel_dep(name = "rules_python", version = "1.0.0")
 
 python = use_extension("@rules_python//python/extensions:python.bzl", "python")
